pages/linux/ip-rule.md

```diff
@@ -0,0 +1,36 @@
+# ip rule
+
+> IP routing policy database management.
+> More information: <https://man7.org/linux/man-pages/man8/ip-rule.8.html>.
+
+- Display the routing policy:
+
+`ip rule {{show|list}}`
+
+- Add a new rule based on packet source addresses:
+
+`sudo ip rule add from {{192.168.178.2/32}}`
+
+- Add a new rule based on the destination address of the specified package:
```

Comment:
The same change should be applied to these other 3 examples too.

Reply:
Done